### PR TITLE
Feature mt smart list primary identifier

### PR DESCRIPTION
**Pull request by migueltarga**

### CODE REVIEW

**Comment by rmarchandTen24** [comment]
was the existing function spelled Aliase?

File: org/Hibachi/HibachiSmartList.cfc [hunk 1]
```diff
@@ -744,7 +744,7 @@ component accessors="true" persistent="false" output="false" extends="HibachiObj
 	}
 	
 	public string function getBaseEntityPrimaryAliase() {
-		return "#variables.entities[ getBaseEntityName() ].entityAlias#.#getService("hibachiService").getPrimaryIDPropertyNameByEntityName(getBaseEntityName())#";
+		return "#variables.entities[ getBaseEntityName() ].entityAlias#.id";
```
Comment (gregmoser) [inline org/Hibachi/HibachiSmartList.cfc]: This is not right.  The primaryKeyPropertyIdentifier is not 'id'

Comment (migueltarga) [inline org/Hibachi/HibachiSmartList.cfc]: I didn't change the function name. 

@gregmoser: https://docs.jboss.org/hibernate/stable/core.old/reference/en/html/queryhql-identifier-property.html

Comment (gregmoser) [inline org/Hibachi/HibachiSmartList.cfc]: I see what your talking about in the documentation.  Can we go over this tomorrow maybe if you have time?

Comment (migueltarga) [inline org/Hibachi/HibachiSmartList.cfc]: Sounds good, I will schedule some time for today, if you are available.

**Comment by gregmoser** [comment]
This looks awesome!  Merge at will!!!

**Comment by rmarchandTen24** [comment]
whoo-ah

**Pull request merged**